ui/src/components/JobForm.vue

```diff
@@ -3,14 +3,13 @@
     <p>
       {{ operation.description }}
     </p>
-    <hydra-operation-form
-      :operation="operation"
-      :resource="resource"
-      :shape="shape"
-      :error="error"
-      :is-submitting="isSubmitting"
-      @submit="onSubmit"
-      :show-cancel="false"
+    <cc-hydra-operation-form
+      :operation.prop="operation"
+      :resource.prop="resource"
+      :shape.prop="shape"
+      :error.prop="error"
+      :submitting.prop="isSubmitting"
```

Comment:
The name is-submitting is more evident than `:resource.prop`

Why the change?

Reply:
Because vue insisted on keeping the kebab syntax as `is-submitted` and not `isSubmitted` as the property is called. the binding did not work that way

Reply:
state returning whatever the form needs to communicate.

Reply:
Ok, I see the comment update now and maybe I did not understand the question...

Reply:
I took that idea from: https://vueuse.org/core/useAsyncState/